### DISCUSSION TITLE
22000.348

### DISCUSSION THREAD

**Discussion by Neo2070** [valinet/ExplorerPatcher#462] [General]
Hi,

I used Windows Update to update Windows 11 on my laptop and it is running 22000.348.  However, this build broke the latest version of ExplorerPatcher, and ExplorerPatcher no longer work.  The Properties no longer show up.  Has anyone tried to test ExplorerPatcher and get it to work with 22000.348?
<img width="470" alt="2021-11-24 15_29_47-" src="https://user-images.githubusercontent.com/94928756/143194058-be445f5d-1271-492d-965f-2416cbe54104.png">

**Comment by BraINstinct0** [comment]
The current latest release support ExplorerPatcher. Restart and give it some time.
Besides, now the default Properties location is the right-click menu of the taskbar, not the start menu button.
That option can be added afterwards.
(Usually downloading symbols take some time.)
If that 'not working' is accompanied by any notification/toast, please share it too; most of the info is already given in those notes.

**Reply by Neo2070** [reply]
Are you also running 22000.348?

**Reply by BraINstinct0** [reply]
Yes. Besides, could you also share your ExplorerPatcher version?

**Reply by Neo2070** [reply]
Not sure what happened. But now ExplorerPatcher works now. I have not done anything since last night.  The version I used is the latest release version I download from this site last 12 hours ago.  I guess it works like you said, "give it some time".  Thank you.

**Reply by BraINstinct0** [reply]
No worries 😉 Good to know the issue is solved!

**Comment by BraINstinct0** [comment]
See https://github.com/valinet/ExplorerPatcher/issues/440

**Comment by valinet** [comment]
Please refer to this material I have published as well, it mainly describes the behavior regarding symbols which will be introduced in the next version which will be published soon: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

Thanks
